### ISSUE TITLE
Impossible to enter text in the "Replace in project" field

### ISSUE THREAD

**Issue by johnbillion** [atom/find-and-replace#949]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: http://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

It has become impossible to enter text into the "Replace in project" field (cmd+shift+f). When doing so, the Find panel closes immediately, and the contents of the field behaves strangely after re-opening the panel.

I've reproduced this both with all my extensions disabled, and in Safe Mode.

### Steps to Reproduce

1. Highlight some text and press cmd+shift+f to open the "Find in project" panel.
2. Tab to the "Replace in project" field and attempt to enter some text.
3. Observe that the panel closes immediately after entering the first character.

**Expected behavior:**

It's expected that the Find panel stays open until the user hits enter, or clicks the "Find All" or "Replace All" button.

**Actual behavior:**

The Find panel closes immediately after entering the first character into the "Replace in project" field.

Upon re-opening the Find panel, the "Replace in project" field continues to behave strangely, allowing only one character to be entered, despite showing the previously entered character.

**Reproduces how often:**

Every time.

### Versions

OS: macOS 10.12.3
Atom    : 1.19.3
Electron: 1.6.9
Chrome  : 56.0.2924.87
Node    : 7.4.0
apm  1.18.4
npm  3.10.10
node 6.9.5 x64
python 2.7.10
git 2.13.3

**Comment by Ben3eeE** [comment]
This happens when the `Close Project Find Panel After Search` setting is enabled. This has been fixed and is currently scheduled for release when Atom 1.20 reaches stable.


---

Thanks for taking the time to contribute!

We noticed that this is a duplicate of https://github.com/atom/find-and-replace/issues/612. This is scheduled to be fixed when Atom 1.20 reaches stable.

Because we treat our issues list as [the Atom team's backlog](https://en.wikipedia.org/wiki/Scrum_(software_development)#Product_backlog), we close duplicates to focus our work and not have to touch the same chunk of code for the same reason multiple times. This is also why we may mark something as duplicate that isn't an exact duplicate but is closely related.

For information on how to use GitHub's search feature to find out if something is a duplicate before filing, see [the How Can I Contribute? section of the Atom CONTRIBUTING guide](https://github.com/atom/atom/blob/master/CONTRIBUTING.md#how-can-i-contribute).
